homeassistant/components/barry/manifest.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "barry",
+  "name": "Barry",
+  "documentation": "https://www.home-assistant.io/integrations/barry/",
+  "dependencies": [],
+  "codeowners": ["@yash", "@ThomasBLarsen"],
```

Same as above. Should be your GitHub handle.
```suggestion
  "codeowners": ["@YashYKhatri", "@ThomasBLarsen"],
```